library/src/test/java/com/chuckerteam/chucker/internal/support/JsonConverterTest.kt

```diff
@@ -31,20 +32,8 @@ class JsonConverterTest {
     }
 
     @Test
-    fun testGsonConfiguration_willUseLowerCaseWithUnderscores() {
-        val json = JsonConverter.instance.toJson(NamingTestClass("aCamelCaseString"))
-        assertThat(json).isEqualTo(
-            """
-            {
-              "a_long_name_with_camel_case": "aCamelCaseString"
-            }
-            """.trimIndent()
-        )
-    }
-
-    @Test
-    fun testGsonConfiguration_willSerializeNulls() {
-        val json = JsonConverter.instance.toJson(NullTestClass(null))
+    fun testMoshiConfiguration_willSerializeNulls() {
+        val json = JsonConverter.instance.adapter(NullTestClass::class.java).addConvenienceMethods().toJson(NullTestClass(null))
```

Comment:
With these changes you just made this test useless.
For Gson there was `serializeNulls()` in `JsonConverter` and this test checked that such configuration worked.
Currently you are manually adding `serializeNulls()` with your `addConvenienceMethods()` right inside this test, so you are not testing `JsonConverter` configuration, but your `addConvenienceMethods()` function.

Reply:
Removed this test case from here.
Added all `MoshiUtils` related test cases in `MoshiUtilsTest` which will test both the methods.
- `getHttpHeaderListJsonAdapter()`
- `addConvenienceMethods()`